docs/metrics/geometry-matching.md

```diff
@@ -209,22 +209,22 @@ unannotated on that image, this inference is excluded in the matching results.
 
 <p style="text-align: center; color: gray;">
     An example of non-exhaustive image-level labeling from
-	<a href="https://storage.googleapis.com/openimages/web/evaluation.html">Open Image Challenge Evaluation</a>
+	<a href="https://storage.googleapis.com/openimages/web/evaluation.html">Open Images Challenge Evaluation</a>
 </p>
 
 The second difference is with handling `group-of` boxes, which is similar to `iscrowd` annotation from
-[COCO](#coco-detection-challenge) but is not just simply ignored. If at least one inference is inside the `group-of`
+[COCO](#coco) but is not just simply ignored. If at least one inference is inside the `group-of`
 box, then it is considered to be a match. Otherwise, the `group-of` box is considered as an unmatched ground truth.
 Also, multiple correct inferences inside the same `group-of` box still count as a single match:
 
 ![An example of group-of boxes](../assets/images/metrics-matcher-oid-group-of.jpg)
 
 <p style="text-align: center; color: gray;">
 	An example of group-of boxes from
-	<a href="https://storage.googleapis.com/openimages/web/evaluation.html">Open Image Challenge Evaluation</a>
+	<a href="https://storage.googleapis.com/openimages/web/evaluation.html">Open Images Challenge Evaluation</a>
 </p>
```

Comment:
This should probably use `<figure>` and `<figcaption>` — see other images in Metrics Glossary for examples